indico/web/client/js/react/forms/fields.jsx

```diff
@@ -117,7 +117,7 @@ export function ReduxCheckboxField({input: {value, ...input}, ...props}) {
     return (
         <ReduxFormField input={input}
                         {...props}
-                        checked={value}
+                        checked={!!value}
```

Comment:
:eyes:  (have to check if there's a better way to get `false` here even if there's no way to put that value in the form's initialValues)